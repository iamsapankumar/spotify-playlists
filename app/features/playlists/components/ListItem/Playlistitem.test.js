/* eslint-disable no-undef */
import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import PlaylistItem from './Playlistitem';

it('renders correctly', () => {
  renderer.create(<PlaylistItem />);
});
describe('PlaylistItem', () => {
  describe('Rendering', () => {
    it('should match to snapshot =>', () => {
      const component = shallow(<PlaylistItem />);
      expect(component).toMatchSnapshot();
    });
  });
});
