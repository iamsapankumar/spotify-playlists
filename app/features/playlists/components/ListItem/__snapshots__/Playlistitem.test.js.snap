// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlaylistItem Rendering should match to snapshot => 1`] = `ShallowWrapper {}`;
